web/docs/cli.md

```diff
@@ -70,7 +71,9 @@ To setup Bash completion, execute `wasp completion` and follow the instructions.
    ```
  
  - `wasp build` generates full web app code, ready for deployment. Use when deploying or ejecting. Generated code goes in the .wasp/build folder.
-  
+
+ - `wasp deploy` makes it easy to get your app hosted on the web. Currently, Wasp offers support for [Fly.io](https://fly.io). Want another hosting provider? Let us know in Discord or make a PR by updating [this TypeScript app](https://github.com/wasp-lang/wasp/tree/release/waspc/packages/deploy)!
```

Comment:
This points to the release branch, just making sure that's what we want.

Reply:
Yeah, I tend to just point there by default so all the docs point to release (even though this isn't there yet).

Reply:
Ah yeah sorry totally missed the intent. Yes, I think for contributor notes `main` probably does make more sense. Will change it 👍🏻

Reply:
Hmmmmm yeah, for the package deploy they probably should be looking at main?